.pytest_cache/README.md

```diff
@@ -0,0 +1,8 @@
+# pytest cache directory #
```

Comment:
Did you add the  .pytest_cache to .gitignore ?

If you already have, run the following command to remove it from the cache: `git rm --cached  .pytest_cache`

Reply:
Done as requested.